ang/civicase/ActivityIcon.html

```diff
@@ -0,0 +1,20 @@
+<span class="civicase__activity-icon" {{activity.icon}}>
+  <span>
+    <span
+      ng-if="!activity.icon && activity.category.indexOf('milestone') < 0"
+      ng-class="{'text-danger': activity.is_overdue}"
+      class="civicase__activity-no-icon">
+      {{activity.type[0]}}
+    </span>
+    <i
+      ng-if="activity.icon"
+      ng-class="{'civicase__activity-icon--ribbon': activity.category.indexOf('milestone') > -1, 'text-danger': activity.is_overdue}"
+      class="fa {{activity.icon}}"></i>
+
+    <i
+      ng-if="direction"
+      ng-class="{'text-danger': activity.is_overdue}"
+      class="civicase__activity-icon-arrow fa fa-arrow-{{ direction }}"></i>
+  </span>
+  <div ng-if="activity.category.indexOf('milestone') > -1" ng-class="{'text-danger': activity.is_overdue}" class="civicase__activity-icon-ribbon"></div>
```

Comment:
`<div>` should not be wrapped inside `<span>`. Use `<i>`or `<span>` tag. Inline elements shouldn't wrap block elements.